File: 13Feb/conditionalStatements.cpp
#include <iostream>
using namespace std;
string thermostat(int temp, int low, int high){

    string state = "";
    if (temp < low){
        state = "Turn on heater";
    }
    else if(temp > high){
        state = "Turn on cooler";
    }
    else{
        state = "Do nothing";
    }
    return state;
}
int main() {
    string state = "";
    state = thermostat(25,20,27);
    cout << "" << state;
    return 0;
}

File: 13Feb/conditionalStatements.py
temp = 25

def thermostatRegulator(temp, low =20,high=27):

    assert high > low , "High temp should be lower than low temp"
    if temp < low:
        state = "Turn on heater"
    elif temp > high:
        state= "Turn on cooler"
    else:
        state = "Do nothing"
    print(state)
thermostatRegulator(temp)